frontend/src/components/public_group/PublicGroupWhoWeAre.js

```diff
@@ -28,9 +30,13 @@ export default class PublicGroupWhoWeAre extends React.Component {
 
           {group.members.length ? (
             <div className='PublicGroup-members pt4'>
-              <h3 className='PublicGroup-subtitle mt0 mb2 -ff-sec -fw-light'>{i18n.getString('coreContributors')}</h3>
+              <h3 className='PublicGroup-subtitle mt0 mb2 -ff-sec -fw-light'>{title}</h3>
               <div className='flex flex-wrap justify-center'>
-                {group.members.map((user, index) => <UserCard user={user} key={index} className='m1' i18n={i18n} />)}
+                {group.members.map((user, index) => {
+                  if(group.slug === 'opensource') user.tier = 'collective';
+                  return <UserCard user={user} key={index} className='m1' i18n={i18n} />
```

Comment:
Have a few other things I want to enable too like braces placement, etc. We can bunch them up in a github issue and fix at the same time.